dataactbroker/README.md

```diff
@@ -616,14 +618,16 @@ Example output:
           "error_name": "rule_failed",
           "field_name": "award_financial",
           "occurrences": "11",
-          "rule_failed": "fain, uri"
+          "rule_failed": "fain, uri",
+          "original_label":""
```

Comment:
Is this sample output for the cross file validation errors? Would we expect there to be a value for `original_label` here?